contracts/modules/Checkpoint/Voting/APLCR/AdvancedPLCRVotingCheckpoint.sol

```diff
@@ -570,6 +575,7 @@ contract AdvancedPLCRVotingCheckpoint is AdvancedPLCRVotingCheckpointStorage, Vo
             uint256 count = length -1;
             for (uint256 i = count; i >= 0 && i < count; i--) {
                 Stage currentStage = getCurrentBallotStage(i);
+                // @FIXME shouldn't we be ORing commit and reveal stages, instead?
```

Comment:
sorry, I didn't get what you are trying to say here

Reply:
The statement below is `true` if `currentStage == Stage.COMMIT` and `currentStage == Stage.REVEAL`, so it requires the stage to be both commit and reveal at the same time. That's why I wonder if the `&&` should be changed to `||`.

Reply:
Yeah it should be `||`, how come I left this and it passes the test as well. Thanks @remon-nashid for pointing this out.